services/keycloak/start.sh

```diff
@@ -827,6 +827,18 @@ EOF
   "scopes": ["drushCacheClear:production","drushCacheClear:development"],
   "policies": ["User has access to project","Users role for project is Guest"]
 }
+EOF
+
+    /opt/jboss/keycloak/bin/kcadm.sh create clients/$CLIENT_ID/authz/resource-server/permission/scope --config $CONFIG_PATH -r lagoon -f - <<EOF
+{
+  "name": "Run Drush cron",
+  "type": "scope",
+  "logic": "POSITIVE",
+  "decisionStrategy": "UNANIMOUS",
+  "resources": ["task"],
+  "scopes": ["drushCron:production","drushCron:development"],
+  "policies": ["User has access to project","Users role for project is Guest"]
+}
```

Comment:
we need to move this outside of the `configure_api_client()` function, as `configure_api_client` is only called once (at the very beginning when keycloak starts the very first time)`.

see https://github.com/amazeeio/lagoon/blob/d9875b2a5b1d53c197d18c29b84b6a2b40e3bf81/services/keycloak/start.sh#L1324 as an example how we add additional things to the keycoak api (like upgrade paths)

Reply:
I have adjusted the logic to support an upgrade path for existing installations.